addons/mass_mailing/wizard/mail_compose_message.py

```diff
@@ -82,9 +82,9 @@ def get_mail_values(self, res_ids):
                 }
                 if mail_values.get('body_html') and mass_mail_layout:
                     mail_values['body_html'] = mass_mail_layout.render({'body': mail_values['body_html']}, engine='ir.qweb', minimal_qcontext=True)
-                # propagate exception state to stat when still-born
-                if mail_values.get('state') == 'cancel':
-                    stat_vals['exception'] = fields.Datetime.now()
+                # propagate ignored state to stat when still-born
+                if mail_values.get('state', None) == 'cancel':
```

Comment:
As dict.get() returns None by default you don't have to give a default value here.